django/forms/widgets.py

```diff
@@ -181,6 +181,7 @@ class Widget(six.with_metaclass(MediaDefiningClass)):
     needs_multipart_form = False  # Determines does this widget need multipart form
     is_localized = False
     is_required = False
+    supports_microseconds = True
```

Comment:
I just added a few cases for `HiddenInput` to the test_initial_datetime_values test [here](https://github.com/django/django/commit/2416bed19fd51080d29a6046874a1556666c607f). @claudep, is this roughly what you had in mind?